tools/AzureRM.Example.psm1

```diff
@@ -9,12 +9,16 @@
 $PSDefaultParameterValues.Clear()
 Set-StrictMode -Version Latest
 
-# Import dependencies using required version, if it is allowed
-if ($PSVersionTable.PSVersion.Major -ge 5)
-{
-%STRICT-DEPENDENCIES%
-}
-else
-{
-%DEPENDENCIES%
+%IMPORTED-DEPENDENCIES%
+
+$FilteredCommands = %COMMANDS%
+
+$FilteredCommands | ForEach-Object {
+	$global:PSDefaultParameterValues.Add($_,
+		{
+			$context = Get-AzureRmContext
```

Comment:
What happens if there is no context (the user hasn't logged in yet)?  Wnat to make sure that we are not pushing errors into the Error stream.

Reply:
Added a null check